python/ray/_private/worker.py

```diff
@@ -2194,6 +2194,12 @@ def disconnect(exiting_interpreter=False):
         ray_actor._ActorClassMethodMetadata.reset_cache()
 
 
+def start_import_thread(worker=global_worker):
```

Comment:
nit: add docstring
nit: add assert to make sure this is only called for workers, no IO or spillers?
nit: can the value of `global_worker` ever change? if so, this will leak memory since it will keep reference around as default parameter